source/index.html.md

```diff
@@ -333,64 +333,71 @@ Depending on card scheme and merchant category, the name on the card might be
 
-Many PSPs have a subscription concept for supporting recurring payments.
-The first approved authorization is the initial recurring authorization (also
-known as "first in series"), all later authorizations are called subsequent
-authorizations. Clearhaus supports subscriptions in the form of recurring
-payments.
+There may be an agreed end of the series.
 
 ### Repeatedly reserve money
 
-A recurring payment is made by making an authorization and setting `recurring`
-parameter to `true`. The first recurring payment for a given card could be made
-this way (notice that the amount may be zero):
+A first-in-series recurring payment is made by making an authorization and
+marking it as a `recurring` series. As an example, a first-in-series recurring
+payment could be made this way:
 
 ````shell
 curl -X POST \
   https://gateway.test.clearhaus.com/authorizations \
   -u <your-api-key>:  \
   -d "amount=2050"    \
   -d "currency=EUR"   \
-  -d "recurring=true" \
+  -d "series[type]=recurring"     \
   -d "card[pan]=4111111111111111" \
   -d "card[expire_month]=06"      \
-  -d "card[expire_year]=2022"     \
+  -d "card[expire_year]=2026"     \
   -d "card[csc]=123"              \
-  --data-urlencode "card[3dsecure][v1][pares]=<some-pares-value>" \
+  --data-urlencode "card[3dsecure][v2][rreq]=<some-rreq-value>" \
   -H "Signature: <signing-api-key> RS256-hex <signature>"
 ````
 
 Example response (snippet):
 
 ````json
 {
-    "id": "e3e9d215-6efc-4c0e-b3d7-2226057c6de8",
-    "status": {
-        "code": 20000
-    },
-    "processed_at": "2018-07-09T12:58:56+00:00",
-    "recurring": true
+    "id": "1b722683-92ad-4c6b-85da-e119d550670d",
+    "status": { "code": 20000 }
 }
 ````
 
-This should be followed by a capture except when the amount is `0`.
+This should be followed by a capture.
 
-Subsequent authorizations are made similarly, however, CSC would not be
-included. For 3-D Secure version 1 (see [3-D Secure](#3-d-secure)), PARes would
-not be included. 3-D Secure version 2 has some support for subsequent
-authorizations.
+Subsequent recurring authorizations initiated by the merchant are made
+similarly, however, CSC is not included, and the previous-in-series is
+referenced, e.g.:
 
-An initial recurring authorization can also be made using the `applepay` and
-`mobilepayonline` payment methods; subsequent recurring payments, however, must
-be made using the `card` payment method using the card details of the initial
-recurring authorization.
+````shell
+curl -X POST \
+  https://gateway.test.clearhaus.com/authorizations \
+  -u <your-api-key>:  \
+  -d "amount=2050"    \
+  -d "currency=EUR"   \
+  -d "series[previous][id]=1b722683-92ad-4c6b-85da-e119d550670d" \
+  -d "card[pan]=4111111111111111" \
+  -d "card[expire_month]=06"      \
+  -d "card[expire_year]=2026"     \
+  -H "Signature: <signing-api-key> RS256-hex <signature>"
+````
+
+A first-in-series authorization can also be made using the `applepay`
```

Comment:
I suppose Samsung Pay, Google Pay and MPO as well?

Reply:
Regarding Google Pay and Samsung Pay: Only after https://github.com/clearhaus/gateway-api-docs/pull/69 and https://github.com/clearhaus/gateway-api-docs/pull/103 (resp.) has been merged 🙂

MPO does not allow recurring on card payments — they have MPO subscriptions and that's a different kind of beast:
* https://developer.mobilepay.dk/subscriptions-main
* https://www.mobilepay.dk/erhverv/abonnementer%20og%20fakturering/mobilepay%20subscriptions